verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -114,8 +116,6 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
             let deleteUrl = this.state.url + rowData.id;
             const response = axios.delete(deleteUrl);
             response.then(() => {
-                // Refresh the table data
-                this.state.tableRef.current.onQueryChange();
                 resolve();
```

Comment:
can this be shortened to response.then(resolve)? (same above)

Reply:
Nice! Done.